app/src/main/java/com/github/fribourgsdp/radio/UserProfileActivity.kt

```diff
@@ -112,12 +105,12 @@ class UserProfileActivity : AppCompatActivity() {
         firebaseAuth = FirebaseAuth.getInstance()
         usernameField = findViewById(R.id.username)
         launchSpotifyButton = findViewById(R.id.launchSpotifyButton)
-        logoutButton = findViewById(R.id.logoutButton)
+        logoutButton = findViewById(R.id.homeButton)
         saveChangeButton = findViewById(R.id.saveUserButton)
         usernameInitialText = findViewById(R.id.usernameInitial)
         spotifyStatusText = findViewById(R.id.spotifyStatus)
         googleSignInButton = findViewById(R.id.googleSignInButton)
-        playlistDisplay = findViewById(R.id.playlist_recycler_view)
+        //playlistDisplay = findViewById(R.id.playlist_recycler_view)
```

Comment:
Why are you commenting this line?

Reply:
Because this line became useless due to the new behaviour of the recycler view.